game/static/game/css/board.css

```
@@ -375,13 +375,23 @@ body {
     font-family: 'Consolas', 'Courier New', monospace;
     font-size: 0.82rem;
     line-height: 1.6;
+
+    display: flex;
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)